commit 88948d982bcd165e05967d2a9c8684eb9f9cbd01
Author: Nils Goroll <nils.goroll@uplex.de>
Date:   Wed Mar 20 11:24:33 2019 +0100

    Change the stack overflow test to 128kb stacksize
    
    on ppc64 fedora, the thread_pool_stack minimum is 128kb due to
    sysconf(_SC_THREAD_STACK_MIN) = 131072
    
    It does not harm the test to use a larger stacksize, so we adjust it to
    this requirement for consistency and simplicity

diff --git a/bin/varnishtest/tests/c00057.vtc b/bin/varnishtest/tests/c00057.vtc
index 5118c79a0..be6569d24 100644
--- a/bin/varnishtest/tests/c00057.vtc
+++ b/bin/varnishtest/tests/c00057.vtc
@@ -12,7 +12,7 @@ server s1 {
 varnish v1 \
 	-arg "-p feature=+no_coredump" \
 	-arg "-p vcc_allow_inline_c=true" \
-	-arg "-p thread_pool_stack=48k" \
+	-arg "-p thread_pool_stack=128k" \
 	-vcl+backend {
 	C{
 #include <signal.h>
@@ -27,11 +27,12 @@ void (*accessor)(volatile char *p) = _accessor;
 
 }C
 	sub vcl_recv { C{
+	    const int stkkb = 128;
 	    int i;
-	    volatile char overflow[48*1024];
+	    volatile char overflow[stkkb * 1024];
 
 	    /* for downwards stack, take care to hit a single guard page */
-	    for (i = 47*1024; i >= 0; i -= 1024)
+	    for (i = (stkkb - 1) * 1024; i >= 0; i -= 1024)
 		accessor(overflow + i);
 	    /* NOTREACHED */
 	    sleep(2);
